stormpiper/stormpiper/spa/src/components/bmpForm.tsx

```diff
@@ -218,6 +208,30 @@ export const BMPForm = forwardRef(function BMPForm(props: formProps, ref) {
     return res;
   }
 
+  function _setFieldValue(
+    fieldID: string,
+    fieldObj: any,
+    isCostField: boolean
+  ) {
+    if (getValues(fieldID)) {
+      // console.log(`Retaining ${fieldID} value of ${getValues(fieldID)}`);
+      return getValues(fieldID);
+    }
+    const fallbackDefault: any =
+      fieldObj.type === "string" ? "" : isCostField ? null : 0;
+    const defaultValue: any = fieldObj.default || fallbackDefault;
+
+    if (props.values) {
+      // console.log(
+      //   `Resetting ${fieldID} value to r${
```

Comment:
can we remove new inactive code? new comments are welcome, but new inactive code is confusing, even if it's logging.

Reply:
done